ssl/statem/extensions.c

```diff
@@ -938,3 +957,143 @@ static int final_sig_algs(SSL *s, unsigned int context, int sent, int *al)
 
     return 1;
 }
+
+
+static int final_key_share(SSL *s, unsigned int context, int sent, int *al)
+{
+    if (!SSL_IS_TLS13(s))
+        return 1;
+
+    /*
```

Comment:
You better start this comment block with `/*-` to avoid any future useless reformating ;)